### PR TITLE
Configure nginx to work with socketio

### PR DESCRIPTION
**Pull request by calvh**
- Modify localhost nginx config
- Add nginx config for websockets

### CODE REVIEW

**Comment by calvh** [comment]
Resolves #72

Please test on your local computer for bugs

Refer to
https://github.com/JonasAlfredsson/docker-nginx-certbot/blob/master/docs/advanced_usage.md#local-ca
for instructions on how to test HTTPS with nginx locally

**Review by Rebecca-CYLi — DISMISSED** [review]
Pulled & Tested it, works fine!

**Comment by calvh** [comment]
After spending the whole evening and night testing, I discovered that the `.tech` domains provided by `domain.com` won't allow certbot to successfully complete the acme challenge required to issue SSL certificates. As I couldn't reach @nimra200, I tested it out with another domain that I registered myself: `rockpaperscissors.tech` and confirmed that something is blocking letsencrypt servers from accessing the tokens in the server during the http-01 challenge.

The latest commit contains the change to `rockpaperscissors.duckdns.org`. Since I generated the certs using the `certbot` command manually, I also changed the volume mapping in `docker-compose.yaml` to use the certs located in the host machine.

I have created an issue #113 to investigate the problem with the `.tech` domains and included a link to an explanation I find most plausible.

tldr: everything is working on https://www.rockpaperscissors.duckdns.org

**Review by Rebecca-CYLi — APPROVED** [review]
If it works, deploy to aws. else, we don't have much time for debugging

**Comment by calvh** [comment]
It's already on AWS. The link I provided above points to our instance. 

I SSH-ed and manually pulled the repo and did all my testing on there.

Websockets work too so the only things left to debug are the game itself.
